doc/reference/reference_lua/datetime.rst

```diff
@@ -1012,3 +1036,148 @@ Support for relational operators for ``interval`` objects has been added since :
     ---
     - true
     ...
+
+Leap second
+-----------
+
+Coordinated Universal Time (UTC) is based on International Atomic Time (TAI).
```

Comment:
Honestly, such detailed info about the leap second looks excessive for Tarantool docs. Can we make a short intro about TZDB in general and the leap seconds in particular, give a link to the external resource, and then jump right to the point (which is how all of these apply to Tarantool)?

Reply:
Updated.